src/tech/hlsjs.js

```diff
@@ -95,6 +101,91 @@ class HlsJs {
       console.log('[videojs-mux-kit] Error: browser does not support MSE nor Hls natively');
     }
   }
+
+  setupQualityLevels() {
+    const ql = this.player.qualityLevels();
+
+    let levels;
+
+    const lastIndex = () => {
+      var last;
```

Comment:
```suggestion
      let last;
```

Reply:
ah, oops. This was left-over from when I was copying things into the chrome console.